config/routes.rb

```diff
@@ -12,10 +12,11 @@
   namespace :admin do
     get '/merchants/:merchant_id', to: "merchants#show"
     get '/merchants', to: "merchants#index"
-    get "/merchants/disable/:id", to: "merchants#disable"
-    get "/merchants/enable/:id", to: "merchants#enable"
+    get "/merchants/disable/:id", to: "merchants#disable" # merchant update (or put in a new status controller for merchant and updating there)
+    get "/merchants/enable/:id", to: "merchants#enable" # merchant update (or put in a new status controller for merchant and updating there)
```

Comment:
notes for refactoring routes to be more restful

Reply:
![](https://media.giphy.com/media/14jrpFn7CaKc8M/giphy.gif)